bootstrap.sh

```diff
@@ -117,6 +117,12 @@ install-dokku-from-deb-package() {
   apt-get -qq -y install apt-transport-https
 
   echo "--> Installing docker"
+  if uname -r | grep -q linode; then
+    echo "--> NOTE: Using Linode? Docker might complain about missing AUFS support."
```

Comment:
Pfft, you let me get away with that [two years ago](https://github.com/callahad/dokku/commit/81523b3bd01910d485b20dba255e298d7cff0a33) ;)